core/runtime/jsruntime.rs

```diff
@@ -83,6 +83,7 @@ pub(crate) struct IsolateAllocations {
 pub(crate) struct ManuallyDropRc<T>(ManuallyDrop<Rc<T>>);
 
 impl<T> ManuallyDropRc<T> {
+  #[allow(unused)]
```

Comment:
Keeping to avoid future footgun.

Reply:
What's the future footgun here? :D

Reply:
`ManuallyDrop<Rc<T>>` can be cloned..  but when you drop the clone it doesn't release the refcount 😢

Reply:
... and because ManuallyDrop<T> derefs to T, that means that a "clone" method on T is called, even if that clone method is on the Rc.